README.md

```diff
@@ -6,13 +6,23 @@ Contains Bazel rules to fetch and install Python dependencies from a requirement
 
 In `requirements.txt`
 ```
-cryptography[test, docs]
-boto3
+cryptography[test, docs]==2.8
+boto3==1.9.253
```

Comment:
Without thinking too much about it, I think if the dep's aren't pinned then you don't get deterministic builds?